main.go

```diff
@@ -295,6 +309,37 @@ func wrapMain() error {
 		}
 	}
 
+	if vargs.KubeInject {
+		// Inject istio sidecar proxy into pods
+		for _, manifestPath := range pathArg {
+			manifestBase := filepath.Base(manifestPath)
+			manifestDir := filepath.Dir(manifestPath)
+			manifestExt := filepath.Ext(manifestPath)
+			manifestBaseWithoutExt := strings.Replace(manifestBase, manifestExt, "", 1)
+			// output path would resemble "/tmp/.kube.istio.yml"
+			istioOutputPath := filepath.Join(manifestDir, fmt.Sprintf("%s.istio%s", manifestBaseWithoutExt, manifestExt))
+			// use `--includeIPRanges`
+			if len(vargs.IncludeIPRanges) > 0 {
+				err = runner.Run(vargs.IstioctlCmd, "kube-inject", "--istioNamespace", vargs.IstioNamespace, "--includeIPRanges", vargs.IncludeIPRanges, "--filename", manifestPath, "--output", istioOutputPath)
+			} else {
+				err = runner.Run(vargs.IstioctlCmd, "kube-inject", "--istioNamespace", vargs.IstioNamespace, "--filename", manifestPath, "--output", istioOutputPath)
+			}
+
+			if err != nil {
+				return fmt.Errorf("Error: %s\n", err)
+			}
+			istioInjectedPathArg = append(istioInjectedPathArg, istioOutputPath)
+		}
+
+		// Apply kube-inject'ed Kubernetes configuration files.
+		err = runner.Run(vargs.KubectlCmd, "apply", "--filename", strings.Join(istioInjectedPathArg, ","))
```

Comment:
what exactly is being applied here that then requires the unmodified files to be applied below? I thought the injected files were augmented with the istio containers.

Reply:
ah. so, these aren't the unmodified files.. these are the modified files which are augmented with the istio containers